src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -8304,6 +8304,11 @@ void MusicXmlParserNotations::technical()
             harmonMute();
         } else if (m_e.name() == "hole") {
             hole();
+        } else if (m_e.name() == "tap") {
+            id = (m_e.attribute("hand") == u"left") ? SymId::guitarLeftHandTapping : SymId::guitarRightHandTapping;
```

Comment:
Why not using `const std::vector<XmlStreamReader::Attribute> attributes = m_e.attributes();``, like further up? Esp. here, as it gets used twice?

Reply:
Here we don't read the element's content.

Reply:
To be more precise: we take only one attribute value for figuring out the glyph, but store all attributes for color, placement, etc.